### DISCUSSION TITLE
Link on Navigation card

### DISCUSSION THREAD

**Discussion by robbrownsell** [Legal-and-General/canopy#1169] [General] [answered]
Hello, I'm trying to use this navigation card component:
https://legal-and-general.github.io/canopy/?path=/story/components-card-examples--navigation-card&globals=backgrounds.grid:true

but I'm having an issue easily using the link attribute. I was hoping it would act a bit like `routerLink` where you can preserve the params but this component doesn't allow that. Could this functionality be added to the component to make it easier to work with?

**Comment by elenagarrone** [comment, marked as answer]
This is being resolved by this PR: https://github.com/Legal-and-General/canopy/pull/1171
